components/routing/src/Middleware/Authenticate.php

```diff
@@ -16,8 +16,8 @@ class Authenticate
      */
     public function handle($request, Closure $next, $guard = null)
     {
-        if (! $_SESSION['user']) {
-            return 'Error Authenticate. Please <a href="/login">login</a>';
+        if (! @$_SESSION['logged_in']) {
```

Comment:
I don't like `@`; what are you covering for here? Can you replace it with `isset`?

Reply:
`isset` would fix this, yes. In the event you don't access the protected routes in the correct order, the `logged_in` index won't exist and you'd get a notice.